bin/convert-argv.js

```diff
@@ -6,6 +6,7 @@ const prepareOptions = require("./prepareOptions");
 const webpackConfigurationSchema = require("./webpackConfigurationSchema.json");
 const validateSchema = require("webpack").validateSchema;
 const WebpackOptionsValidationError = require("webpack").WebpackOptionsValidationError;
+const findup = require("findup-sync");
```

Comment:
Is there some other package as alternative? This one is 161.9
 kilobyte minified (https://bundlephobia.com/result?p=findup-sync@2.0.0)

Reply:
@ematipico Yes, this package is large (findup-sync), but this one supports RegExp. I think with this solution the config files would be discovered with equal chance and the code will be more simplified. (I already implemented it, commit: https://github.com/lakatostamas/webpack-cli/commit/710e04a)
I have found an alternative, but it doesn't support RegExp (find-up: https://bundlephobia.com/result?p=find-up@3.0 ) It would be complicated to implement to find config files with an equal chance on every level without RegExp. (but with this PR it would be easy to implement: https://github.com/sindresorhus/find-up/pull/28 )
Which solution do you recommend?
1, Go with the large npm lib which supports RegExp
2, Go with the small npm lib which only accepts a string. In this case, I think we have to implement a solution which stops on every level and tries both patterns.
3, Implement our own solution without a third party (with RegExp)
4, any other solution?

I would like to clarify this before I fix the other issues.

Reply:
Actually this link shows some better results 

https://packagephobia.now.sh/result?p=findup-sync

It looks to be smaller than expected!